code/game/objects/items/plushes.dm

```diff
@@ -461,8 +461,8 @@
 		P.clashing = FALSE
 
 /obj/item/toy/plush/narplush
-	name = "nar'sie plushie"
-	desc = "A small stuffed doll of the elder god Nar'Sie. Who thought this was a good children's toy?"
+	name = "Nar-Sie plushie"
```

Comment:
I think this needs `\improper`, because you'd still talk about the Nar-Sie plushie.